analysis/wwz/wwz4l.py

```diff
@@ -169,17 +169,24 @@ def columns(self):
     def process(self, events):
 
         # Dataset parameters
-        dataset = events.metadata["dataset"]
+        json_name = events.metadata["dataset"]
+
+        isData             = self._samples[json_name]["isData"]
+        histAxisName       = self._samples[json_name]["histAxisName"]
+        year               = self._samples[json_name]["year"]
+        xsec               = self._samples[json_name]["xsec"]
+        sow                = self._samples[json_name]["nSumOfWeights"]
+        if "2022" in year:
```

Comment:
It is a much stronger check and less error prone to do `if year in ["2022", "2022EE"]` rather than `if "2022" in year`